pax/plugins/peak_processing/PeakAreaCorrections.py

```diff
@@ -64,3 +69,223 @@ def transform_event(self, event):
+                    #for sample in range(start_sample,end_sample):
+                    for sample in range(peak_left_edge,peak_right_edge):
+                        if(sample<channel_sat_left[pmtid]):
+                            area_model_left+=wf_sum_nan_saturate[sample-start_sample]
+                            area_data_left+=wf_raw[pmtid,sample-start_sample]   
+                                                        
+                        elif(sample>channel_sat_right[pmtid]):
+                            area_model_right+=wf_sum_nan_saturate[sample-start_sample]
+                            area_data_right+=wf_raw[pmtid,sample-start_sample]
+                            
+                    area_data=area_data_left+area_data_right
+                    area_model=area_model_left+area_model_right
+
+                    # prefer to make corrections based on rising edge, this is possibly helpful for based saturation correction as well.                    
+                    if (channel_sat_status[pmtid]>0) & (area_data_left>10) & (area_model>200) & (area_model_left>100) :
+                        if((area_data_left>10) or (area_data_left/area_data>0.3) ):
+                            area_pmt_correction_factor[pmtid]=area_data_left/area_model_left
+                        # if rising edge is too small, use wfs after saturation
+                        elif (area_data>10):
+                            area_pmt_correction_factor[pmtid]=area_data/area_model
+                        area_pmt_after_correction[pmtid]=area_nan_saturate*area_pmt_correction_factor[pmtid]
+                    # for non saturated channels, update the calculation based on new peak edges
+                    elif(peak_right_edge>peak_left_edge):
+                        area_new=0.0
+                        for sample in range(peak_left_edge,peak_right_edge):
+                            area_new+=wf_raw[pmtid,sample-start_sample]
+                        area_pmt_after_correction[pmtid]=area_new
+                    
+                    # re_compute the s2 area per channel, based on correction factor and the new edges
+                    #area_pmt_after_correction[pmtid]=area_nan_saturate*area_pmt_correction_factor[pmtid]
+                    peak.area_per_channel[pmtid]=area_pmt_after_correction[pmtid]
+                   
+                    # Re compute area fraction on top
+                    if (pmtid<nPmtsTop):
+                        areatop+=peak.area_per_channel[pmtid]
+                       
+                print('area before correction: ',peak.area)                
+                peak.area=np.sum(peak.area_per_channel)
+                print('area after correction: ',peak.area)
+                
+                #print('aft before correction: ',peak.area_fraction_top)
+                if(peak.area>0):
+                    peak.area_fraction_top=areatop/peak.area
+                    #print('aft after correction: ',peak.area_fraction_top)
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->